packages/discovery-provider/ddl/migrations/0093_splits_in_purchases.sql

```diff
@@ -0,0 +1,63 @@
+-- Recreates the extended purchase splits from the information in the purchases table
+-- Required to populate purchase details modal
+BEGIN;
+
+ALTER TABLE
+    usdc_purchases
+ADD
+    COLUMN IF NOT EXISTS splits JSONB;
+
+WITH relevant_timestamps AS (
+    SELECT
+        user_id,
+        MAX(block_timestamp) AS block_timestamp,
+        signature
+    FROM
+        user_payout_wallet_history
+        JOIN usdc_purchases ON seller_user_id = user_id
+    WHERE
+        block_timestamp < usdc_purchases.created_at
```

Comment:
should this be <=? maybe in practice it doesn't matter

Reply:
Yeah I don't think this matters. Also I'd argue the price change being processed strictly before the purchase makes more sense anyway